style-gutenburg.scss

```diff
@@ -18,4 +18,9 @@ Text Domain: mojblocks
     @import 'node_modules/govuk-frontend/govuk/settings/media-queries';
     @import 'node_modules/govuk-frontend/govuk/helpers/media-queries';
     @import 'src/**/editor.scss';
-}
\ No newline at end of file
+}
+
+/// adjust width of editor to match NHS theme
+body.block-editor-page.kt-editor-width-default div.wp-block {
+    max-width: 960px;
+}
```

Comment:
Not sure the plugin should be influencing the theme editor (even if it is in the backend)?

Reply:
Thanks mate. Removed for now without an additional solution. The editor styling is now confusing but we can change that.